scripts/figma-import/icons.js

```diff
@@ -0,0 +1,81 @@
+const config = require('../../figma-import.config')
+
+const path = require('path')
+const fs = require('fs').promises
+const { pascalCase } = require('change-case')
+
+const { getImageContent, getNodeChildren, getSvgImageUrl } = require('./utils/api')
+
+const IconsDir = path.resolve(__dirname, '../../src/shared/icons/figmaSvgs')
+
+const getIconFolderPath = () => path.resolve(IconsDir)
+
+let counter = 0
+
+/**
+ * clear icons dir
+ *
+ */
+const clearIconsDir = async () => {
+  try {
+    await fs.rm(IconsDir, { recursive: true })
+    console.log(`${IconsDir} successfully deleted!`)
+  } catch (err) {
+    console.error(`Error while deleting ${IconsDir}.`)
+  }
+}
+
+/**
+ * generate icon content
+ * [iconName].svg files
+ *
+ * @param {object} iconNode
+ * @param {number} total
+ * @return {Promise<void>}
+ */
+const generateIcon = async (iconNode, total) => {
+  const iconUrl = await getSvgImageUrl(iconNode.id)
+
+  const iconName = pascalCase(iconNode.name)
```

Comment:
I think we could stick to `kebabCase` names like we have in `src/shared/icons/svgs`. 
We also don't need `change-case` library, we could just use `kebabCase` from `lodash`

Reply:
Unfortunately I wasn't able to use `lodash-es` because of CommonJS usage in this script. `require() of ES modules is not supported.`. Do you think we can live with that? I think it much less effort than changing everything to ES modules.

Reply:
hmm, I guess I'm fine with using it

Reply:
Oh, sorry, consulted this with Klaudiusz. Let's just use `lodash`( the one without `-es`). I'm using it as well in `sd.config.js` and it works.